<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTrans</class>
 <widget class="QWidget" name="CTrans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEdit_p1">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>25</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p2">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>25</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p3">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>25</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p4">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>25</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p5">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>25</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p6">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>25</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="pushButton_ChoseFile">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>27</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pathDisplay">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>27</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::Direction::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_BeginTrans">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_PauseTrans">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>暂停发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_StopTrans">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>停止发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
